### PR TITLE
Add Error Handling

### PR DESCRIPTION
**Pull request by Infiniticity**

### CODE REVIEW

**Review by vsmart-06 — CHANGES_REQUESTED** [review]
I think you have to add `global logs` in both the command error functions.

**Comment by Infiniticity** [comment]
@vsmart-06 I don't think so, because the `global` keyword is only required if you are changing the variable.
Here, it is just being accessed.

**Comment by vsmart-06** [comment]
Alright then

**Review by vsmart-06 — CHANGES_REQUESTED** [review]
You have to change the `ctx.author.mention` to `ctx.user.mention` in line 51.

I also tested this code myself and the `traceback.format_exception` needs 2 arguments and I don't know what they are so I would suggest that you just send the exception directly as it looks fine that way too.

File: app.py [hunk 1]
```diff
@@ -41,14 +41,14 @@ async def on_ready():
 
 @bot.event
 async def on_command_error(ctx, exception):
-    description = f"Channel: {ctx.channel.mention}\nUser: {ctx.author.mention}\nGuild: {ctx.guild.name} ({ctx.guild.id})\n\nError:\n```{traceback.format_exception(exception)}```"
+    description = f"Channel: {ctx.channel.mention}\nUser: {ctx.user.mention}\nGuild: {ctx.guild.name} ({ctx.guild.id})\n\nError:\n```{''.join(traceback.format_exception(exception, exception, exception.__traceback__))}```"
```
Comment (vsmart-06) [inline app.py:44]: This part should be `ctx.author` only. The change is valid only for application commands.

**Review by vsmart-06 — CHANGES_REQUESTED** [review]

**Comment by vsmart-06** [comment]
Everything looks good as long as the `format_exception` part works.

**Comment by Infiniticity** [comment]
Yes, that works.

**Pull request merged**